R/position-stack.r

```diff
@@ -195,7 +195,7 @@ PositionStack <- ggproto("PositionStack", Position,
       )
     }
 
-    rbind(neg, pos)
+    rbind(neg, pos)[match(seq_len(nrow(data)), c(which(negative), which(!negative))),]
```

Comment:
I don't see immediately that this is always correct. Can there be problems with `NA`s? Or will `NA`s never reach this point?

Reply:
An `NA` value in ymax is pretty nonsensical and would also have broken the previous code. I can put in some guarding if you’d like nonetheless

Reply:
No strong opinion either way. Just make sure to look into why the CI builds fail.

Reply:
The failures are due to the downstream changes in sf, which have been fixed in master but not yet merged into the `v3.2.1-rc` branch